config/launcher-macos64.dhall

```diff
@@ -0,0 +1,40 @@
+\(perCluster: { keyType : Text, relays : Text, reportServer : Text, updateServer : Text })
+->
+{ key      = "mainnet_${perCluster.keyType}wallet_macos64"
+, pass     =
+  { reportServer        = perCluster.reportServer
+  , nodePath            = "./cardano-node"
+  , nodeDbPath          = "$HOME/Library/Application Support/Daedalus/DB-1.0"
+  , nodeLogPath         = "$HOME/Library/Application Support/Daedalus/Logs/cardano-node.log"
+  , nodeArgs =
```

Comment:
`nodeArgs` can be in final `launcher.dhall` and then just OS/environment specific settings can be substituted.

Reply:
This is mainly because right now, someone might add nodeArgs here and forget to update the other file.

os/environment specific settings can be a simple record that is at the very end substituted in `launcher.dhall`.